functions/test/endpoints/FirestoreDataFactory.ts

```diff
@@ -332,226 +349,25 @@ export class FirestoreDataFactory{
-    enabled?: boolean,
-    single_use?: boolean
-}
-
-export const LINK_DEFAULTS:LinkOptions = {
-    archived: false,
-    description: "Basic description",
-    enabled: true,
-    single_use: true
-}
-
-/**
- * Default fields for Reward
- */
-export const REWARD_DEFAULTS:RewardOptions = {
-    id: "Pizza Party",
-    required_ppr: 100,
-    required_value: 20000
-}
-
-/**
- * Default fields for point log
- */
-export const POINT_LOG_DEFAULTS:PointLogOptions = {
-    approved_by: "Preapproved",
-    approved_on: new Date(Date.parse("5/18/2020")),
-    date_occurred: new Date(Date.parse("5/18/2020")),
-    date_submitted: new Date(Date.parse("5/18/2020")),
-    description: "Empty Description",
-    floor_id: "4N",
-    point_type_id: 1,
-    point_type_name: "Empty Name",
-    point_type_description: "Empty Description",
-    rhp_notifications: 0,
-    resident_first_name: "TEST_FIRST",
-    resident_last_name: "TEST_LAST",
-    resident_notifications: 0
-
-}
-
-/**
- * Default fields for system preferences
- */
-export const SYSTEM_PREFERENCES_DEFAULTS:SystemPreferenceOptions = {
-    android_version: "2.0.0",
-    one_time_code: "abc",
-    competition_hidden_message: "hidden",
-    house_enabled_message: "Honors1",
-    ios_version: "1.6.2",
-    is_competition_visible: true,
-    is_house_enabled: true,
-    suggested_point_ids: "1,2,3,4",
-}
-
-/**
- * default fields for point type
- */
-export const POINT_TYPE_DEFAULTS:PointTypeOptions = {
-    description: "Empty Point Type Description",
-    name: "Empty Point Type Name",
-    is_enabled: true,
-    permission_level: 2,
-    residents_can_submit: true,
-    value: 1
-}
-
-/**
- * Default field for house
- */
-export const HOUSE_DEFAULTS:HouseOptions = {
-    color: "#5AC0C7",
-    total_points: 20,
-    num_residents: 200
-}
-
-/**
- * Default field for house code
- */
-export const HOUSE_CODE_DEFAULTS:HouseCodeOptions = {
-    code: "4N1234",
-    code_name: "4N Resident",
-    floor_id: "4N",
-    house: "Platinum",
-    permission_level: 0
-}
+    constructor(status: PointStatus){
+        this.status = status
+    }
 
-/**
- * Default fields for user
- */
-export const USER_DEFAULTS:UserOptions = {
-    first: "TEST_FIRST",
-    last: "TEST_LAST",
-    total_points: 0,
-    semester_points: 0,
-    house_name: "Platinum",
-    floor_id: "4N"
+    offset(value:number): PointStatus{
+        this.status.house_points += value
+        this.status.user_points += value
+        this.status.user_semester_points += value
+        return this.status
+    }
 }
 
-/**
- * Default PointLogMessage Options
- */
-export const MESSAGE_DEFAULTS:PointLogMessageOptions = {
-    creation_date: new Date(Date.now()),
-    message: "Empty Message",
-    message_type: "comment",
-    sender_first_name: "First",
-    sender_last_name: "Last",
-    sender_permission_level: 0
-}
+export declare type PointStatus = {
+    user_points:number,
+    user_semester_points:number,
+    house_points:number
+}
```

Comment:
All of the Options have been moved to a separate file named OptionDeclarations.ts for clarity.

CompetitionPointStatus and PointStatus are now defined in this file.